packages/volto/news/6186.feature

```diff
@@ -0,0 +1 @@
+Improved url regex to allow non-public urls, such as https://intranet/ or file://server/share, e.g., for intranet. @mamico
```

Comment:
We need to wrap URLs with backticks to make these inline literals, else linkcheckbroken attempts to visit the URL and fails.

```suggestion
Improved URL regex to allow non-public or intranet URLs, such as `https://intranet/` or `file://server/share`. @mamico
```